src/hooks/player_management/assign_physical_index.cpp

```diff
@@ -15,12 +15,12 @@
 
 namespace big
 {
-	inline bool is_spoofed_host_token(uint64_t token)
+	inline bool is_spoofed_host_token(rage::rlGamerInfo* info)
 	{
-		if (token < 200'000'000)
+		if (info->m_host_token < INT_MAX)
 			return true;
 
-		return false;
+		return (info->m_peer_id >> 32) != (info->m_host_token >> 32);
```

Comment:
will this check return false for normal users, the ones who are not using any mod menu

Reply:
yeah sorry mb you can resolve this. You already answered in comment above.